releasenotes/notes/deprecate-span-meta-ad1c96ba55acaac9.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    py:attribute::Span.meta has been deprecated. Use py:method::Span.get_tag(key) and py:method::Span.set_tag(key, val) instead.
```

Comment:
```suggestion
    :py:attr:ddtrace.Span.meta has been deprecated. Use :py:meth:ddtrace.Span.get_tag and :py:meth:ddtrace.Span.set_tag instead.
```

Reply:
Awesome. I'll use that format in my other release notes